lms/envs/aws.py

```diff
@@ -919,3 +919,16 @@
 SEARCH_SKIP_ENROLLMENT_START_DATE_FILTERING = FEATURES.get("SEARCH_SKIP_ENROLLMENT_START_DATE_FILTERING", True)
 #### RaccoonGang ####
 
+if FEATURES.get("ENABLE_RG_INSTRUCTOR_ANALYTICS", False):
+    INSTALLED_APPS += ('rg_instructor_analytics', 'rg_instructor_analytics_log_collector')
+
+    if 'web_fragments' not in INSTALLED_APPS:
+         INSTALLED_APPS += ('web_fragments',)
+
+    RG_ANALYTICS_GRADE_STAT_UPDATE = FEATURES.get("RG_ANALYTICS_GRADE_STAT_UPDATE", {
+        'minute': '*',
```

Comment:
starts at every minute past every 6th hour. May be better set 'minute': '0'? It will be started at minute 0 past every 6th hour.

Reply:
Thanks.
Change to each 15 mins.
Please check.
@OlhaShyliaieva